contrib/ci/fedora.sh

```diff
@@ -5,6 +5,10 @@ set -x
 #get any missing deps from the container
 ./contrib/ci/fwupd_setup_helpers.py install-dependencies --yes -o fedora
 
+dnf install -y \
+    https://kojipkgs.fedoraproject.org//packages/libgusb/0.4.4/1.fc38/x86_64/libgusb-0.4.4-1.fc38.x86_64.rpm \
+    https://kojipkgs.fedoraproject.org//packages/libgusb/0.4.4/1.fc38/x86_64/libgusb-devel-0.4.4-1.fc38.x86_64.rpm
```

Comment:
I don't want to be merging it like this, can we land it first?

Reply:
Sure, of course; I've got to wait a few days to get it in to the stable channel: https://bodhi.fedoraproject.org/updates/FEDORA-2023-33ab83a659